### ISSUE TITLE
Deploy fails with "The filename, directory name, or volume label syntax is incorrect."

### ISSUE THREAD

**Issue by blairconrad** [FakeItEasy/FakeItEasy#1576]
When [releasing 5.1.0](https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/builds/22274755#L430):

```
ushing FakeItEasy.5.1.0.nupkg
The filename, directory name, or volume label syntax is incorrect.
Unhandled Exception: SimpleExec.NonZeroExitCodeException: The process exited with code 1.
   at SimpleExec.ProcessExtensions.Throw(Process process) in C:\projects\simple-exec\SimpleExec\ProcessExtensions.cs:line 36
   at SimpleExec.Command.<RunAsync>d__1.MoveNext() in C:\projects\simple-exec\SimpleExec\Command.cs:line 57
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at FakeItEasy.Deploy.Program.<UploadPackageToNuGetAsync>d__2.MoveNext() in C:\projects\fakeiteasy\tools\FakeItEasy.Deploy\Program.cs:line 92
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at FakeItEasy.Deploy.Program.<Main>d__0.MoveNext() in C:\projects\fakeiteasy\tools\FakeItEasy.Deploy\Program.cs:line 63
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at FakeItEasy.Deploy.Program.<Main>(String[] args)
Command exited with code -532462766
```

@thomaslevesque suggests that it's due to https://github.com/adamralph/simple-exec/issues/112 and that we can just update to 5.0.1.

**Comment by blairconrad** [comment]
Building had been succeeding, but deployment failed. FakeItEasy.Build doesn't use quoted parameters in the calls to SimpleExec, but FakeItEasyDeploy does, so @thomaslevesque's assessment sounds good.

**Comment by blairconrad** [comment]
This change has been released as part of [FakeItEasy 5.1.1](https://github.com/FakeItEasy/FakeItEasy/releases/tag/5.1.1).
